packages/button/src/LionButton.js

```diff
@@ -1,28 +1,37 @@
 import { css, html, DelegateMixin, SlotMixin } from '@lion/core';
+import { DisabledMixin } from '@lion/field';
 import { LionLitElement } from '@lion/core/src/LionLitElement.js';
 
-export class LionButton extends DelegateMixin(SlotMixin(LionLitElement)) {
+export class LionButton extends DisabledMixin(DelegateMixin(SlotMixin(LionLitElement))) {
   static get properties() {
     return {
-      disabled: {
-        type: Boolean,
-        reflect: true,
-      },
       role: {
         type: String,
         reflect: true,
       },
-      tabindex: {
-        type: Number,
-        reflect: true,
-      },
     };
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  _beforeTemplate() {
+    return html`
+      <slot name="icon-before"></slot>
+    `;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  _afterTemplate() {
+    return html`
+      <slot name="icon-after"></slot>
+    `;
+  }
```

Comment:
we will override it in the `select-invoker` to introduce the arrow down

Reply:
we agreed to keep these templates empty by default (just return a blank lit-html template in both), because we just need a hook in the template, not slots themselves

Reply:
done